portfolio/src/main/webapp/style.css

```diff
@@ -11,7 +11,7 @@ body {
 }
 
 #fullpage {
-  min-width: 100vw;
+min-width: 100vw;
```

Comment:
indent by 2 spaces and swap with `min-height:` below to maintain sorted order.

Reply:
Updated with commit d133fce